worker/workercmd/worker.go

```diff
@@ -265,5 +265,5 @@ func (cmd *WorkerCommand) baggageclaimRunner(logger lager.Logger) (ifrit.Runner,
 
 	cmd.Baggageclaim.OverlaysDir = filepath.Join(cmd.WorkDir.Path(), "overlays")
 
-	return cmd.Baggageclaim.Runner(nil)
+	return cmd.Baggageclaim.Runner(nil, cmd.Containerd.PrivilegedMode)
```

Comment:
> worker/workercmd/worker.go:268:53: cmd.Containerd.PrivilegedMode undefined (type ContainerdRuntime has no field or method PrivilegedMode)

cmd.Containerd.PrivilegedMode is undefined when building on different OS (darwin/windows)

Reply:
Nothing different happens with the garden backend. This is a containerd specific setting. Everything defaults to the current status quo.